smoke_tests.sh

```diff
@@ -2,3 +2,4 @@
 set -ex
 
 [ $(curl --write-out %{http_code} --silent --output /dev/null https://$(hostname)/) == 200 ]
+[ $(curl --write-out %{http_code} --silent --output /dev/null https://$(hostname)/api/v2/articles) == 200 ]
```

Comment:
/admin
/explorer
both of these should exist but both load the login page as they are protected

rss feeds should exist:
/rss/articles/poa+vor/last-28-days/